contributions/api/contributions_rest_service.py

```diff
@@ -38,6 +39,9 @@
 app.add_api('contribution.yaml', base_path=cfg.CONTRIBUTION_URL_PREFIX, arguments={'title': 'Rokwire'}, resolver=RokwireResolver('controllers'),
             resolver_error=501, strict_validation=True)
 
+# App is behind one proxy that sets the -For and -Host headers.
```

Comment:
Hi, @minump, As is an issue with the Catalog, this piece of code needs to be moved to the Catalog. Thanks.